superset-frontend/src/dashboard/components/SliceHeader/SliceHeader.test.tsx

```diff
@@ -356,7 +356,7 @@ test('Should render "annotationsError"', () => {
   render(<SliceHeader {...props} />, { useRedux: true, useRouter: true });
   expect(
     screen.getByRole('img', {
-      name: 'One ore more annotation layers failed loading.',
+      name: 'One or more annotation layers failed loading.',
```

Comment:
New spell-check-this rule:
https://github.com/check-spelling/spell-check-this/blob/b968ca32e0e5961a85c7fe3e9431f81553c3f434/.github/actions/spelling/line_forbidden.patterns#L67-L68